### ISSUE TITLE
Enable ticket_fields import

### ISSUE THREAD

**Issue by sminnee** [singer-io/tap-freshdesk#24]
The Ticket Fields API - /api/v2/ticket_fields - provides important information such as the look-up values for ticket statuses.

Docs here: https://developers.freshdesk.com/api/#list_all_ticket_fields

It would be great if this tap included ticket_fields as well.

**Comment by briansloane** [comment]
Agreed. This would be a valuable addition to the tap. If you can submit a PR we'll review and merge.
